style.css

```diff
@@ -2,33 +2,879 @@ html {
+    margin-left: 135px;
+    margin-right: 24px;
+    margin-top: 130px;
+  }
+
+  .contact-form {
+    display: flex;
+    flex-direction: column;
+    align-items: left;
+    border: solid;
+    border: none;
+    flex: 1;
+    margin-top: 130px;
+    margin-bottom: 230px;
+  }
+
+  .my-ul {
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .name-li {
+    border-bottom-style: solid;
+    border-bottom-width: 1px;
+    border-color: #dbd8d7;
+    margin-bottom: 36px;
+    flex: 1;
+    padding-bottom: 12px;
+    margin-right: 172px;
+  }
+
+  .name-input {
+    width: 584px;
+    height: 18px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #979493;
+    border: none;
+    text-align: left;
+    padding-left: 24px;
+  }
+
+  .email-li {
+    border-bottom-style: solid;
+    border-bottom-width: 1px;
+    border-color: #dbd8d7;
+    margin-bottom: 24px;
+    flex: 1;
+    padding-bottom: 12px;
+    margin-right: 172px;
+  }
+
+  .email-input {
+    width: 584px;
+    height: 18px;
+    left: 16px;
+    right: 16px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #979493;
+    border: none;
+    text-align: left;
+    padding-left: 24px;
+  }
+
+  .text-area {
+    height: 114px;
+    width: 584px;
+    border: none;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #3c3a39;
+    background: #fbf8f7;
+    margin-right: 4px;
+    padding-left: 24px;
+  }
+
+  .get-in-touch-form {
+    height: 48px;
+    width: 137px;
+    padding: 12px;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    background: #ff6b00;
+    color: white;
+    border: none;
+    margin-left: 40px;
+  }
+
+  .get-in-touch-form:hover {
+    height: 48px;
+    width: 137px;
+    padding: 12px;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    text-align: center;
+    letter-spacing: 0.03em;
+    background: #fff5e1;
+    color: #ff6b00;
+    border: none;
+  }
+
+  .wrapper-B {
+    width: 1440px;
+    height: 44px;
+    background-color: white;
+    display: flex;
+    justify-items: right;
+    border-top-style: solid;
+    border-top-color: #dfe1e6;
+    border-top-width: 1px;
   }
 
-  a {
+  .social-group-end {
+    width: 307px;
+    height: 24px;
+    background-color: white;
     color: white;
-    text-decoration: none;
+    justify-items: space-around;
+    gap: 30px;
+    display: flex;
+    align-items: center;
+    margin-left: 600px;
+    padding-top: 12px;
   }
+}
 
+@media only screen and (max-width: 768px) {
```

Comment:
- [ ] I kindly suggest that you make your project styling mobile-first. That would mean that you need to use `min-width` to style for desktops and not `max-width` to style for mobile.

Reply:
Dear Topeogunleye, I made the changes I consider pertinent. But one not for the following reason:
If I use min-width (769px)  it means to apply the CSS format for a screen with more than 769px (desktop ).
Conversely, if I use max-width (768) it will render the mobile CSS style  
for a screen with less than 768px (mobile) as it was done in the code. So no change is required.

Reply:
@edahigure, I'd mark this as optional but kindly note that the project is supposed to be a mobile-first approach, as changing it now would make you change a lot of code which you may not meet with the deadline, but do implement the changes later. :pray:

Reply:
@edahigure, that's understandable